### PR TITLE
Added Default URL for apps

### LINKED ISSUE #214: Default URL for apps
Currently when you click on VIEW APP button on https://account.codingblocks.com/apps/ you're redirected to a concatenated string. Instead we want to have a column named default URL which will save the default URL of the app to which we want to send the user.

### PR DESCRIPTION
**Pull request by architkshk**
closes #214 @witty123 Please review
screenshot from 2018-06-15 15-54-17
.
screenshot from 2018-06-15 21-30-56

screenshot from 2018-06-15 21-30-33

### CODE REVIEW

**Comment by architkshk** [comment]
Added screenshots

File: src/routers/pages/client.js [hunk 1]
```diff
@@ -7,6 +7,18 @@ const acl = require('../../middlewares/acl')
 
 const models = require('../../db/models').models
 
+router.get('/me',
```
Comment (championswimmer) [inline src/routers/pages/client.js]: remove this. it has moved to `/users/me/clients`

Comment (architkshk) [reply src/routers/pages/client.js]: Resolved.

**Comment by architkshk** [comment]
@championswimmer Done!

**Comment by witty123** [comment]
@championswimmer I think we should keep defaultUrl not null for maintaining sanity. What say ?

**Comment by architkshk** [comment]
@championswimmer @witty123 any updates?

**Pull request merged**